.github/workflows/pull_request.yaml

```diff
@@ -93,18 +79,15 @@ jobs:
         if: matrix.env.TEST_TYPE == 'release-upgrade' || matrix.env.TEST_TYPE == 'source-upgrade'
         run: |
           docker pull quay.io/pulp/pulp-ci-dbuster:3.0.0
-          docker pull quay.io/pulp/pulp_rpm-ci-f31:3.1.0
-          docker pull quay.io/pulp/pulp_rpm-ci-c7:3.1.0
+          docker pull quay.io/pulp/pulp-ci-c7:3.0.0
+          docker pull quay.io/pulp/pulp-ci-f31:3.0.0
       - name: Pulling images for distro packages upgrades
         if: matrix.env.TEST_TYPE == 'packages-upgrade'
         run: |
           docker pull quay.io/pulp/pulp-ci-pkgs-c7:3.3.1
           docker pull quay.io/pulp/pulp-ci-pkgs-c8:3.3.1
       - name: Run tox
-        run: |
-          tox && exit 0 || true
-          sleep 45
-          tox
+        run: tox
```

Comment:
:+1: we should definitely accept this change.
Where did this even come from?
If the installer is regularly failing on the same task (i guess package install), we should do the retry in ansible with that specific task. It even helps to make users happier.

Reply:
it came from the time we were learning github actions,
and it was generally pulling docker failures